libs/ledger/benchmark/state_sentinel_bench.cpp

```diff
@@ -40,7 +38,7 @@ void StateSentinelAdapter_BasicBenchmark(benchmark::State &state)
   BitVector shards{1};
   shards.SetAllOne();
 
-  StateSentinelAdapter adapter{storage, Identifier{"foo.bar"}, shards};
+  StateSentinelAdapter adapter{storage, {"foo.bar"}, shards};
```

Comment:
`ConstByteArray` has implicit constructor I would therefore drop the braces so that it is at least consistent with other parts of the code base.

```suggestion
  StateSentinelAdapter adapter{storage, "foo.bar", shards};
```